core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -82,13 +82,8 @@ class TestRaftServer(
       () => Features.fromKRaftVersion(MetadataVersion.MINIMUM_KRAFT_VERSION))
     socketServer = new SocketServer(config, metrics, time, credentialProvider, apiVersionManager)
 
-    val metaProperties = MetaProperties(
-      clusterId = Uuid.ZERO_UUID.toString,
-      nodeId = config.nodeId
-    )
-
     raftManager = new KafkaRaftManager[Array[Byte]](
-      metaProperties,
+      Uuid.ZERO_UUID.toString,
```

Comment:
Do we want `Uuid.randomUuid.toString` instead, which is what we use in `RaftManagerTest`?

Reply:
It's a good question. I don't know why this test chose `Uuid.ZERO_UUID`. But I figured changing it here was out of scope.